lib/snowpack/type_parser.ex

```diff
@@ -40,11 +42,16 @@ defmodule Snowpack.TypeParser do
 
   defp parse({:datetime, data}), do: DateTimeParser.parse_datetime!(data)
 
+  defp parse({:float, data}) when is_float(data), do: data
+
   defp parse({:float, data}) when is_binary(data) do
-    {float, ""} = Float.parse(data)
-    Decimal.from_float(float)
+    case Float.parse(data) do
+      {float, _rest} -> Decimal.from_float(float)
+      :error -> return_raw(:float, data, :float_parse_error)
+    end
```

Comment:
Handle errors in parsing. Log and return the original value. Probably, better than blowing up.